src/templates/profile.ts

```diff
@@ -15,3 +16,53 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
+"""
+usecase ${name} {
+  input {
+    "field title"
+    foo! string 
+
+    bar 
+  }
+
+  result {
+    baz string
+  }
+
+  error issue
```

Comment:
I know Z mentioned the `error error` problem, but I'd personally prefer simply `error Error` - it's already in use in station (`social-media/post-lookup`, `chat/channels`, `chat/threads`...) and `issue` is a bit overloaded.

Alternately, `error ErrorModel` (`social-media/profiles`).

It's a bit Java-y and redundant, but IMHO more fitting